tests/test_client.py

```diff
@@ -52,41 +51,48 @@ def test_update_with_invalid_content_type(client):
 
 
 # ----------------------------------------------------------------------------------------------------
-def test_update_with_valid_content_type(client):
-    for content_type in ["all", "org", "markdown", "image", "pdf", "notion", "plugin1"]:
-        # Act
-        response = client.get(f"/api/update?t={content_type}")
-        # Assert
-        assert response.status_code == 200, f"Returned status: {response.status_code} for content type: {content_type}"
-
-
-# ----------------------------------------------------------------------------------------------------
-def test_update_with_github_fails_without_pat(client):
+def test_regenerate_with_invalid_content_type(client):
     # Act
-    response = client.get(f"/api/update?t=github")
+    response = client.get(f"/api/update?force=true&t=invalid_content_type")
 
     # Assert
-    assert response.status_code == 500, f"Returned status: {response.status_code} for content type: github"
-    assert (
-        response.json()["detail"]
-        == "🚨 Failed to update server via API: Github PAT token is not set. Skipping github content"
-    )
+    assert response.status_code == 422
 
 
 # ----------------------------------------------------------------------------------------------------
-def test_regenerate_with_invalid_content_type(client):
+def test_index_batch(client):
+    # Arrange
+    request_body = {
+        "org": {"additionalProp1": "string", "additionalProp2": "string", "additionalProp3": "string"},
+        "pdf": {"additionalProp1": "string", "additionalProp2": "string", "additionalProp3": "string"},
+        "plaintext": {"additionalProp1": "string", "additionalProp2": "string", "additionalProp3": "string"},
+        "markdown": {"additionalProp1": "string", "additionalProp2": "string", "additionalProp3": "string"},
```

Comment:
Should we populate the test data with something closer to what we expect in the request body? Such tests help understand the code better IMO.

See e.g below
```suggestion
        "org": {"/path/to/file1.org": "content of file 1 ", "/path/to/file2.org": "content of file 2", "/path/to/file3.org": "content of file 3"},
        "pdf": {"additionalProp1": "string", "additionalProp2": "string", "additionalProp3": "string"},
        "plaintext": {"additionalProp1": "string", "additionalProp2": "string", "additionalProp3": "string"},
        "markdown": {"additionalProp1": "string", "additionalProp2": "string", "additionalProp3": "string"},
```